commandapi-platforms/commandapi-bukkit/commandapi-bukkit-test/commandapi-bukkit-test-tests/src/test/java/dev/jorel/commandapi/test/arguments/ArgumentMultiLiteralTests.java

```diff
@@ -86,6 +86,62 @@ void commandBuildingTestWithMultiLiteralArgument() {
 				{
 				  "type": "root",
 				  "children": {
+				    "commandapitest:command1": {
+				      "type": "literal",
+				      "children": {
+				        "option1": {
+				          "type": "literal",
+				          "children": {
+				            "option1": {
+				              "type": "literal",
+				              "executable": true
+				            },
+				            "option2": {
+				              "type": "literal",
+				              "executable": true
+				            },
+				            "option3": {
+				              "type": "literal",
+				              "executable": true
+				            }
+				          }
+				        },
+				        "option2": {
+				          "type": "literal",
+				          "children": {
+				            "option1": {
+				              "type": "literal",
+				              "executable": true
+				            },
+				            "option2": {
+				              "type": "literal",
+				              "executable": true
+				            },
+				            "option3": {
+				              "type": "literal",
+				              "executable": true
+				            }
+				          }
+				        },
+				        "option3": {
+				          "type": "literal",
+				          "children": {
+				            "option1": {
+				              "type": "literal",
+				              "executable": true
+				            },
+				            "option2": {
+				              "type": "literal",
+				              "executable": true
+				            },
+				            "option3": {
+				              "type": "literal",
+				              "executable": true
+				            }
+				          }
+				        }
+				      }
+				    },
```

Comment:
We could perhaps specify the `minecraft` namespace when testing argument structure like here so that we don't get these duplicates listed in the json, which aren't relevant to what we're testing and just take up a lot of space. We could also just leave it; https://github.com/CommandAPI/CommandAPI/pull/602 modifies the json dump to indicate cases like this better, where an argument tree structure shows up in multiple places.